releases/3.0.16.md

```diff
@@ -0,0 +1,53 @@
+---
+title: TiDB v3.0.16 Release Notes
+category: Releases
+aliases: ['/docs/dev/releases/v3.0.16/']
+---
+
+# TiDB v3.0.16 Release Notes
+
+Release date: July 01, 2020
+
+TiDB version: v3.0.16
+
+## Bug Fixes
+
++ TiDB
+
+    - Fix read/write inconsistent result when meet lock that point to a primary key has be insert/delete in own txn [#18248](https://github.com/pingcap/tidb/pull/18248)
+    - Fix "Got too many pings" grpc error log in PD-server follower [#17944](https://github.com/pingcap/tidb/pull/17944)
+    - Fix panic when the child of HashJoin returns TypeNull column. [#17935](https://github.com/pingcap/tidb/pull/17935)
+    - Correct the usage of json.Unmarshal in job.DecodeArgs to be compatible with future Go versions。 [#17887](https://github.com/pingcap/tidb/pull/17887)
+    - Discard feedbacks generated from `delete` / `update` statements. [#17841](https://github.com/pingcap/tidb/pull/17841)
+    - Fix error message when access denied. [#17722](https://github.com/pingcap/tidb/pull/17722)
+    - Fix JSON comparison for int and float. [#17715](https://github.com/pingcap/tidb/pull/17715)
+    - Failpoint: update failpoint which will cause data race before [#17710](https://github.com/pingcap/tidb/pull/17710)
+    - Fix pre-split region timeout constraint not work when create table. [#17617](https://github.com/pingcap/tidb/pull/17617)
+    - Fix panic after sending fail with ambiguous errors [#17378](https://github.com/pingcap/tidb/pull/17378)
+    - Planner: support `is null` filter condition in hash partition pruning [#17308](https://github.com/pingcap/tidb/pull/17308)
+    - Fix flashback table failed in some special cases. [#17165](https://github.com/pingcap/tidb/pull/17165)
+    - Util: fix the wrong result when where stmt only have string column [#16658](https://github.com/pingcap/tidb/pull/16658)
+    - Fix error of query when `only_full_group_by` SQL mode is set [#16620](https://github.com/pingcap/tidb/pull/16620)
+    - Fix wrong return length for case when function [#16562](https://github.com/pingcap/tidb/pull/16562)
+
+## Others
```

Comment:
@uglyengineer Please confirm the notes below belong to which category.

Reply:
@hicqu @qw4990 @disksing  Is 'others ' category right? Should these be sorted to 'bug fixes', 'new features', 'new changes', 'Improvements' or 'Compatibility Changes'? Or these shouldn't be here?

Reply:
I think `Improvements` is better.

Reply:
oh, not LGTM.

Reply:
PD's change should be in bug fixes.

Reply:
fixed.